### ISSUE TITLE
Why does this exist

### ISSUE THREAD

**Issue by jb55** [tadfisher/android-nixpkgs#92]
Why would I use this over androidComposition in nixpkgs? If this is better in some way, why not upstream it to nixpkgs?

**Comment by irisjae** [comment]
https://github.com/NixOS/nixpkgs/blob/3f00b40fff0e6de822196dd926d8efca430b490b/pkgs/development/mobile/androidenv/compose-android-packages.nix#L30

composeAndroidPackages in nixpkgs only supports x86.

**Comment by mackattack3k** [comment]
> composeAndroidPackages in nixpkgs only supports x86.

I don't think this is the case since https://github.com/NixOS/nixpkgs/pull/304716 anymore

**Comment by irisjae** [comment]
Oh that's amazing! I may have to reconsider my choice of packages some time.

**Comment by adrian-gierakowski** [comment]
even though compose-android-packages.nix in nixpkgs can now be built on `aarch64-darwin` it produces a broken emulator since it outputs an x86_64 binary instead of arm64 one

emulator produced using code from this repo produces arm64 binary correctly
